canonical-web-and-design/ubuntu.com
# Added github username to contributor agreement form

## Done

- Added github username to contributor agreement form per request from legal
- Updated the [copy doc](https://docs.google.com/document/d/1MfgTzlAxiFE6VUW1qAPmrks2ui1zzsX5T7d-DZJlOAw/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/legal/contributors/agreement
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the form submits and there are no errors
